### ISSUE TITLE
The result of code and change the train data

### ISSUE THREAD

**Issue by idmakers** [phil-bergmann/tracking_wo_bnw#61]
I have ran the code on the UBUNTU 16.04 by anaconda.
And I run the  script "test_tracker.py"
But the result is always zero like the below
https://imgur.com/2dbBicu
the tracker found is zero 

And how do I change to my own train data?

**Comment by ddwell** [comment]
I think I am having the same issue. 

`INFO - main - Initializing object detector.
INFO - main - Tracking: MOT17-02-FRCNN
100%|█████████████████████████████████████████| 600/600 [00:26<00:00, 22.79it/s]
INFO - main - Tracks found: 0
INFO - main - Runtime for MOT17-02-FRCNN: 26.3 s.
INFO - main - Writing predictions to: /home/vladimir/Projects/materials/tracking_wo_bnw/output/tracktor/MOT17/Tracktor++
INFO - main - Tracking: MOT17-04-FRCNN
100%|███████████████████████████████████████| 1050/1050 [00:45<00:00, 22.84it/s]
INFO - main - Tracks found: 0
INFO - main - Runtime for MOT17-04-FRCNN: 46.0 s.
`

**Comment by timmeinhardt** [comment]
Please double check if you have the MOTChallenge dataset files in the correct directories. In the past, this was usually the issue for this error.

**Comment by ddwell** [comment]
> Please double check if you have the MOTChallenge dataset files in the correct directories. In the past, this was usually the issue for this error.

Apologies for causing disturbances.

This has begun when I tried to inference Your solution.
Blindly I managed to run the code on my own video with no errors. like this:
`blob = {
            'dets': torch.Tensor(mot_boxes).unsqueeze(0),
            'img': torch.tensor( np.transpose(frame,(2,0,1))*(1/255) ).float().unsqueeze(0),
            'scores': torch.Tensor(nscores)
        }
        tracker.step( blob )`

but `tracker.get_results() ` was giving an empty list [ ]

Then I tried to run the provided script `experiments/scripts/test_tracktor.py` and it gave me 0 tracks found for each of the MOT17-XX-FRCNN

I think I have MOT17 downloaded with all data, labels and detections, in the right directories

**Comment by idmakers** [comment]
Thank for the reply .
I change the path  and it can run

**Comment by timmeinhardt** [comment]
Perfect. Hopefully the repository helps you with your project. Can we close this issue?
